### ISSUE TITLE
 [OUDS] Create component - Badge

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/Orange-Boosted-Bootstrap#2898]
# Description
**As a** user of the Ouds library
**I want** to be able to use the component
**So that** I can add an Ouds Badge component in my own projects

**As a** user of the library
**I want** to be able to use the Badge component
**So that** I have access to the documentation on how to use it and
* I can see examples of implementations with the available states and customizations
* I can copy and paste the code of the selected customization for reuse

# Context 
The component is described in the Design tools and allows to describe the various states and variants.
## Variants
TBD

## Demo
TBD

**Comment by B3nz01d** [comment]
Here are the feedbacks I have on this component:
* I see that the badge in the accent status is Orange in the Figma design and is yellow in the web implementation
* I'm not sure of this but I think in the icon badge the icon positioning does not have the right padding as the heart icon seems to be too close to the border of the badge

**Comment by nilloq** [comment]
> [@nilloq](https://github.com/nilloq) Here are the feedbacks I have on this component:
> 
> * I see that the badge in the accent status is Orange in the Figma design and is yellow in the web implementation
> * I'm not sure of this but I think in the icon badge the icon positioning does not have the right padding as the heart icon seems to be too close to the border of the badge

@B3nz01d 
* the accent status will come when we will integrate the next token revision 1.2
* I confirm we have the right padding, the issue is the same I raised when we reviewed the chevron link. The icon in Figma is slightly different from the Solaris one (either not the good one taken by desing or re-build by Figma...)
